_posts/2021-01-26-converting-puppetized-dsc-modules.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "Converting to use Puppetized DSC Modules"
+author: pmcmaw
+categories:
+  - dsc
+  - puppet
+  - modules
+tags:
+  - tooling
+  - dsc resources
+  - windows
+  - modules
+  - forge
+  - powershell gallery
+---
+
+## Why bother?
+
+The main reason we are advocating moving to this new and shiny method is that it provides you with a long list of advantages and new features. 
```

Comment:
I think we'll probably want to link to/reference our announcement blog as a way to short circuit some of this explanation (and can maybe quote from it?):

https://puppetlabs.github.io/iac/news/roadmap/2020/09/21/dsc-release.html